numismatic/cli.py

```diff
@@ -10,6 +10,8 @@
 from streamz import Stream
 import attr
 
+from numismatic.feeds import *
```

Comment:
Is there a way to import into another namespace instead of the global namespace when using `from ... import *`?

Reply:
I just tried to do this inside the Exchange.factory() method but got an error saying that * imports are only allowed at the module level.

Reply:
Interestingly, you can do this: `exec('from ... import *')` - that won't throw errors, but I don't know what namespace it uses because I can't find the imported things.